warehouse/ingest-core/src/test/resources/datawave/ingest/mapreduce/job/IngestJob-test-type.xml

```diff
@@ -19,7 +19,7 @@
   <value>ingest-test</value>
   <description>This is the type of data being ingested.</description>
 </property>
-
+<!--
```

Comment:
Lets get @keith-ratcliffe to look at this one

Reply:
Turns out this was un-necessary. Must have been something else broken while I was in the middle of making changes. I've removed this change.